DependencyInjection/Configuration.php

```diff
@@ -269,6 +269,10 @@ private function addViewSection(ArrayNodeDefinition $rootNode)
 
     private function addBodyListenerSection(ArrayNodeDefinition $rootNode)
     {
+        $decodersDefaultValue = ['json' => 'fos_rest.decoder.json'];
+        if (class_exists('FOS\RestBundle\Decoder\XmlDecoder')) {
```

Comment:
please add a `use` statement for this class and make use of the `class` constant here